### ISSUE TITLE
[che-operator] Operator installer failed being run by chectl with default value of che version

### ISSUE THREAD

**Issue by dmytro-ndp** [eclipse-che/che#16072]
### Describe the bug
There was **che-operator** deployment error when it was tried to install _Eclipse Che_ on **minikube** by command:

> chectl server:start --k8spodreadytimeout=180000 --installer=operator --listr-renderer=verbose --platform=minikube --che-operator-cr-yaml=custom-resource.yaml

Che operator pod logs:
> time="2020-02-19T01:39:04Z" level=fatal msg="Failed to initialize default value: 'CHE_VERSION'. Environment variable with default value was not found." 

<details>
 <summary>Chectl logs</summary>
 <pre>
[10:55:16] Verify Kubernetes API [started]
[10:55:16] Verify Kubernetes API...OK [title changed]
[10:55:16] Verify Kubernetes API...OK [completed]
[10:55:16] 👀  Looking for an already existing Eclipse Che instance [started]
[10:55:16] Verify if Eclipse Che is deployed into namespace "che" [started]
[10:55:16] Verify if Eclipse Che is deployed into namespace "che"...it is not [title changed]
[10:55:16] Verify if Eclipse Che is deployed into namespace "che"...it is not [completed]
[10:55:16] 👀  Looking for an already existing Eclipse Che instance [completed]
[10:55:16] ✈️  Minikube preflight checklist [started]
[10:55:16] Verify if kubectl is installed [started]
[10:55:16] Verify if kubectl is installed [completed]
[10:55:16] Verify if minikube is installed [started]
[10:55:16] Verify if minikube is installed [completed]
[10:55:16] Verify if minikube is running [started]
[10:55:16] Verify if minikube is running [completed]
[10:55:16] Start minikube [started]
[10:55:16] Start minikube [skipped]
[10:55:16] → Minikube is already running.
[10:55:16] Check Kubernetes version [started]
[10:55:17] Check Kubernetes version: Found v1.14.3. [title changed]
[10:55:17] Check Kubernetes version: Found v1.14.3. [completed]
[10:55:17] Verify if minikube ingress addon is enabled [started]
[10:55:17] Verify if minikube ingress addon is enabled [completed]
[10:55:17] Enable minikube ingress addon [started]
[10:55:17] Enable minikube ingress addon [skipped]
[10:55:17] → Ingress addon is already enabled.
[10:55:17] Retrieving minikube IP and domain for ingress URLs [started]
[10:55:17] Retrieving minikube IP and domain for ingress URLs...10.0.101.55.nip.io. [title changed]
[10:55:17] Retrieving minikube IP and domain for ingress URLs...10.0.101.55.nip.io. [completed]
[10:55:17] ✈️  Minikube preflight checklist [completed]
Eclipse Che logs will be available in '/tmp/chectl-logs/1582041316045'
[10:55:17] Start following logs [started]
[10:55:17] Start following Eclipse Che logs [started]
[10:55:17] Start following Eclipse Che logs...done [title changed]
[10:55:17] Start following Eclipse Che logs...done [completed]
[10:55:17] Start following Postgres logs [started]
[10:55:17] Start following Postgres logs...done [title changed]
[10:55:17] Start following Postgres logs...done [completed]
[10:55:17] Start following Keycloak logs [started]
[10:55:17] Start following Keycloak logs...done [title changed]
[10:55:17] Start following Keycloak logs...done [completed]
[10:55:17] Start following Plugin registry logs [started]
[10:55:17] Start following Plugin registry logs...done [title changed]
[10:55:17] Start following Plugin registry logs...done [completed]
[10:55:17] Start following Devfile registry logs [started]
[10:55:17] Start following Devfile registry logs...done [title changed]
[10:55:17] Start following Devfile registry logs...done [completed]
[10:55:17] Start following logs [completed]
[10:55:17] Start following events [started]
[10:55:17] Start following namespace events [started]
[10:55:17] Start following namespace events...done [title changed]
[10:55:17] Start following namespace events...done [completed]
[10:55:17] Start following events [completed]
[10:55:17] 🏃‍  Running the Che Operator [started]
 ›   Warning: Eclipse Che will be deployed in Multi-User mode as 'operator' 
 ›   installer supports only that mode.
[10:55:17] Copying operator resources [started]
[10:55:17] Copying operator resources...done. [title changed]
[10:55:17] Copying operator resources...done. [completed]
[10:55:17] Create Namespace (che) [started]
[10:55:17] Create Namespace (che)...done. [title changed]
[10:55:17] Create Namespace (che)...done. [completed]
[10:55:17] Create ServiceAccount che-operator in namespace che [started]
[10:55:17] Create ServiceAccount che-operator in namespace che...done. [title changed]
[10:55:17] Create ServiceAccount che-operator in namespace che...done. [completed]
[10:55:17] Create Role che-operator in namespace che [started]
[10:55:17] Create Role che-operator in namespace che...done. [title changed]
[10:55:17] Create Role che-operator in namespace che...done. [completed]
[10:55:17] Create ClusterRole che-operator [started]
[10:55:17] Create ClusterRole che-operator...done. [title changed]
[10:55:17] Create ClusterRole che-operator...done. [completed]
[10:55:17] Create RoleBinding che-operator in namespace che [started]
[10:55:17] Create RoleBinding che-operator in namespace che...done. [title changed]
[10:55:17] Create RoleBinding che-operator in namespace che...done. [completed]
[10:55:17] Create ClusterRoleBinding che-operator [started]
[10:55:17] Create ClusterRoleBinding che-operator...done. [title changed]
[10:55:17] Create ClusterRoleBinding che-operator...done. [completed]
[10:55:17] Create CRD checlusters.org.eclipse.che [started]
[10:55:17] Create CRD checlusters.org.eclipse.che...done. [title changed]
[10:55:17] Create CRD checlusters.org.eclipse.che...done. [completed]
[10:55:17] Waiting 5 seconds for the new Kubernetes resources to get flushed [started]
[10:55:22] Waiting 5 seconds for the new Kubernetes resources to get flushed...done. [title changed]
[10:55:22] Waiting 5 seconds for the new Kubernetes resources to get flushed...done. [completed]
[10:55:22] Create deployment che-operator in namespace che [started]
[10:55:22] Create deployment che-operator in namespace che...done. [title changed]
[10:55:22] Create deployment che-operator in namespace che...done. [completed]
[10:55:22] Create Eclipse Che Cluster eclipse-che in namespace che [started]
[10:55:22] Create Eclipse Che Cluster eclipse-che in namespace che...done. [title changed]
[10:55:22] Create Eclipse Che Cluster eclipse-che in namespace che...done. [completed]
[10:55:22] 🏃‍  Running the Che Operator [completed]
[10:55:22] ✅  Post installation checklist [started]
[10:55:22] PostgreSQL pod bootstrap [started]
[10:55:22] scheduling [started]
[11:00:27] scheduling [failed]
[11:00:27] → ERR_TIMEOUT: Timeout set to pod wait timeout 300000. podExist: false, currentPhase: undefined
[11:00:27] PostgreSQL pod bootstrap [failed]
[11:00:27] → ERR_TIMEOUT: Timeout set to pod wait timeout 300000. podExist: false, currentPhase: undefined
[11:00:27] ✅  Post installation checklist [failed]
[11:00:27] → ERR_TIMEOUT: Timeout set to pod wait timeout 300000. podExist: false, currentPhase: undefined
 ›   Error: Error: ERR_TIMEOUT: Timeout set to pod wait timeout 300000. 
 ›   podExist: false, currentPhase: undefined
 ›   Installation failed, check logs in '/tmp/chectl-logs/1582041316045'
 </pre>
</details>

custom-resource.yaml content: https://gist.github.com/dmytro-ndp/91ad3f3c61fba6fbd939d21d585af682

### Related commit
https://github.com/eclipse/che-operator/commit/e9c5d3f312f07d73187075fcdd36423aae374edc

### Che version

 - [ ] latest
 - [x] nightly
 - [ ] other: please specify

### Steps to reproduce
1. Install minikube 1.1.1.
2. Download chectl from _next_ channel.
3. Execute the command:
```
chectl server:start --k8spodreadytimeout=180000 --installer=operator --listr-renderer=verbose --platform=minikube --che-operator-cr-yaml=custom-resource.yaml
```

### Expected behavior
chectl installs Eclipse Che of nightly version

### Runtime
   - [ ] kubernetes (include output of `kubectl version`) 
   - [ ] Openshift (include output of `oc version`)
   - [x] minikube 1.1.1 (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [x] chectl:next with operator installer: chectl/0.0.20200218-next.3cc1458 linux-x64 node-v10.19.0
   - [ ] che-operator 
   - [ ] minishift-addon
   - [ ] I don't know

### Environment
   - [ ] my computer 
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify) 
   - [x] other: CRW CCI

### Additional context
It affects Eclipse Che / Che-Theia PR check.

**Comment by tolusha** [comment]
Pls attach CR file

**Comment by dmytro-ndp** [comment]
@tolusha: here it is https://gist.github.com/dmytro-ndp/91ad3f3c61fba6fbd939d21d585af682

**Comment by tolusha** [comment]
`chectl` released with dependencies on latest `che` and `che-operator` templates.

**Comment by nickboldt** [comment]
Was this fix backported to 7.9.x branch? I'm wondering how it was done (I don't see a linked PR).

Was it this?

https://github.com/eclipse/che-operator/commit/b001254b022513ed6eb332647857e20750cb41cf

And if so how does that fix the 7.9.x branch? Should I just run it and see what happens?

**Comment by tolusha** [comment]
https://github.com/eclipse/che/issues/16206
Create build directory in the root of the project and copy Dockerfile there.
